### PR TITLE
remove msdn library

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #2037

### CODE REVIEW

File: docs/framework/deployment/deployment-guide-for-developers.md [hunk 1]
```diff
@@ -353,11 +353,11 @@ The following table lists options that you can include when you chain the .NET F
...
 |**/log** `file` &#124; `folder`|Specifies the location of the log file. The default is the temporary folder for the process, and the default file name is based on the package. If the file extension is .txt, a text log is produced. If you specify any other extension or no extension, an HTML log is created.|
 |**/msioptions**|Specifies options to be passed for .msi and .msp items; for example: `/msioptions "PROPERTY1='Value'"`.|
-|**/norestart**|Prevents the setup program from rebooting automatically. If you use this option, the chaining app has to capture the return code and handle rebooting (see [Getting Progress Information from an Installation Package](https://go.microsoft.com/fwlink/?LinkId=179606) in the MSDN Library).|
+|**/norestart**|Prevents the setup program from rebooting automatically. If you use this option, the chaining app has to capture the return code and handle rebooting (see [Getting Progress Information from an Installation Package](https://docs.microsoft.com/previous-versions/cc825975(v=vs.100))).|
```
Comment (Thraka) [inline docs/framework/deployment/deployment-guide-for-developers.md]: One too many `)` in the url. 😄 

```suggestion
|**/norestart**|Prevents the setup program from rebooting automatically. If you use this option, the chaining app has to capture the return code and handle rebooting (see [Getting Progress Information from an Installation Package](https://docs.microsoft.com/previous-versions/cc825975(v=vs.100)).|
```

**Review by Thraka — APPROVED** [review]
Just one bug to fix

File: docs/framework/deployment/deployment-guide-for-developers.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/deployment/deployment-guide-for-developers.md]: The link is actually inside parentheses so I think that’s correct

Comment (mairaw) [reply docs/framework/deployment/deployment-guide-for-developers.md]: 0D1E01A9-2049-45BE-8AF8-1F63C56CCFC0

**Pull request merged**